examples/launchpad/launchpad.py

```diff
@@ -0,0 +1,31 @@
+from polygon import RESTClient
+from polygon.rest.models.request import RequestOptionBuilder
+
+
+def get_list_trades_launchpad():
+    client = RESTClient()
+
+    """
+    set headers example:
+    options = RequestOptionBuilder()
+        .edge_headers(edge_id="EDGE_ID", edge_ip_address="EDGE_ID_ADDRESS")
+        
+    update headers example:
+    options = options.update_edge_header(edge_ip_address="NEW_IP")
+    """
```

Comment:
here's the change; I can add edge_user to line 11.
is there somewhere else you wanted this implemented?

This is in response to your last comment that had been `resolved`